### DISCUSSION TITLE
Unable to fetch the front-end Node in Prometheus

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1631] [Q&A] [answered]
Hi,
Unable to fetch the front-end Node in Prometheus, After adding the tag to frontend instance. 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/c471b1a9-aa98-4f3e-ae05-9bd2fdbb3b1e)
Prometheus Public IP: 54.237.205.12

**Comment by sreedharm07** [comment]
hi @omswetha 
in your /opt/prometheus/**prometheus.yml**  file you dint added any node exporter ip there is local host so its showing only local host prometheus status 
please configure node exporter info in prometheus.yml file which was in path  /opt/prometheus/

**Reply by omdevops99** [reply, marked as answer]
I have updated in intellij tool but it is not reflecting in the file. what would be the issue ?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/eed9b20c-665b-426a-93aa-1196a7466140)
